compiler/rustc_ast/src/ast.rs

```diff
@@ -1446,6 +1446,9 @@ pub enum ExprKind {
     /// with an optional value to be returned.
     Yeet(Option<P<Expr>>),
 
+    /// Bytes included via `include_bytes!`
```

Comment:
Could you add comments explaining why this variant was added (optimization) and that it should always behave like `ExprKind::Lit` with a byte string literal?